### PR TITLE
Bumps `nimble v0.16.4`

### PR DESCRIPTION
**Pull request by jmgomez**

### CODE REVIEW

**Comment by jmgomez** [comment]
@Araq dont merge yet please. Need to update `nimble` one more time..

**Comment by narimiran** [comment]
> @Araq dont merge yet please. Need to update `nimble` one more time..

Can it be merged now?

**Comment by jmgomez** [comment]
> > @Araq dont merge yet please. Need to update `nimble` one more time..
> 
> Can it be merged now?

Yes, please

File: tools/deps.nim [hunk 1]
```diff
@@ -42,3 +42,9 @@ proc cloneDependency*(destDirBase: string, url: string, commit = commitHead,
     discard "this dependency was bundled with Nim, don't do anything"
   else:
     quit "FAILURE: " & destdir & " already exists but is not a git repo"
+
+proc updateSubmodules*(dir: string) =
+  let oldDir = getCurrentDir()
+  setCurrentDir(dir)
+  exec "git submodule update --init"
```
Comment (Araq) [inline tools/deps.nim:49]: In some follow-up PR use `try finally` here.

**Pull request merged**